package.json

```diff
@@ -26,5 +26,10 @@
     "turbo": "^1.10.15",
     "typescript": "^5.2.2"
   },
+  "pnpm": {
+    "overrides": {
+      "@auth/core": "0.18.0"
+    }
+  },
```

Comment:
shouldn't be necessary when nextauth5 is properly released, rn types conflict without this override since beta has a peerdep on experimental auth/core which isn't always up-to-date with main